examples/mugen/retrieval/video_clip.py

```diff
@@ -23,6 +24,8 @@ class TextEncoder(nn.Module):
     Args:
         pretrained (bool): whether to use a pretrained model or not.
             Defaults to True.
+        trainable (bool): whether the encoder's weights should be trainable (not frozen).
+            Defaults to True. Ignored if ``pretrained`` is False.
```

Comment:
Would add a warning statement for "no-surprise" reason, e.g.,  when a user says `pretrained = False` and `trainable = False` but instead the behavior is `trainable = True`.